typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -95,6 +97,7 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     transactions.push(
       ...(await this.updateIsm(actualConfig, expectedConfig)),
       ...(await this.updateRemoteRouters(actualConfig, expectedConfig)),
+      ...(await this.transferOwnership(actualConfig, expectedConfig)),
```

Comment:
```suggestion
      ...this.updateRemoteRouters(actualConfig, expectedConfig),
      ...this.updateOwnership(actualConfig, expectedConfig),
```

name change for consistency; removed awaits bc not async

Reply:
Fixed